README.md

```diff
@@ -1,50 +1,62 @@
-# template-server
+# Textures Helper
 
-## Architecture
+Textures helper compress textures using DXT1 mipmapped format.
 
-Extension of "ports and adapters architecture", also known as "hexagonal architecture".
+## Converting an image
 
-With this architecture, code is organized into several layers: logic, controllers, adapters, and components (ports).
+This service exposes and endpoint in charge of compressing textures which expects two inputs `size` as a path parameter and `asset` as a query parameter. Below can be found the call structure to this endpoint:
 
-## Application lifecycle
+```
+GET <HOST>/content/dxt/:LENGTH?asset=<ASSET-URL>
+```
+
+* **length** - must be an integer of a power of two that can go from 128 until 2048
+* **asset** - must be a HTTP url aiming to the asset to be converted by the service
 
-1. **Start application lifecycle** - Handled by [src/index.ts](src/index.ts) in only one line of code: `Lifecycle.run({ main, initComponents })`
-2. **Create components** - Handled by [src/components.ts](src/components.ts) in the function `initComponents`
-3. **Wire application & start components** - Handled by [src/service.ts](src/service.ts) in the funciton `main`.
-   1. First wire HTTP routes and other events with [controllers](#src/controllers)
-   2. Then call to `startComponents()` to initialize the components (i.e. http-listener)
+The response structure will look like:
 
-The same lifecycle is also valid for tests: [test/components.ts](test/components.ts)
+```
+{
+  "asset": string,
+  "conversionRatio":
+    {
+      "height": number,
+      "width": number
+     }
+}
+```
 
-## Namespaces
+* **asset** - contains the URL of the resized and compressed asset. This URL will aim to the CDN configured for this service.
+* **conversionRatio** - contains the resize ratio relating old asset size with the new one specified when calling the endpoint
+  * **height** - newHeight / oldHeight (_fixed to four decimals_)
```

Comment:
Being a `number`, does this need to be rounded? In general rounding applies for string types...

Reply:
I'll check this, if confirmed that this is not needed I'll update the code and this part of the documentation in a new PR.

Thank you!